fuel/converters/iris.py

```diff
@@ -0,0 +1,71 @@
+import os
+
+import h5py
+import numpy
+
+from fuel.converters.base import fill_hdf5_file
+
+
+def convert_iris(directory, output_directory, output_filename='iris.hdf5'):
+    """Converts the Iris dataset to HDF5.
+    Converts the Iris dataset to an HDF5 dataset compatible with
+    :class:`fuel.datasets.Iris`. The converted dataset is
+    saved as 'iris.hdf5'.
+    This method assumes the existence of the file `iris.data`.
+    It assumes the existence of the following file:
+    * `iris.data`
+    Parameters
+    ----------
+    directory : str
+        Directory in which input files reside.
+    output_directory : str
+        Directory in which to save the converted dataset.
+    output_filename : str, optional
+        Name of the saved dataset. Defaults to `None`, in which case a name
+        based on `dtype` will be used.
+    dtype : str, optional
+        Either 'float32', 'float64', or 'bool'. Defaults to `None`,
+        in which case images will be returned in their original
+        unsigned byte format.
```

Comment:
I think the `dtype` description is a leftover from some copy-paste.

As a matter of fact, do we really need this much flexibility, or would it be okay to use `float64` and add `ForceFloatX` as a default transformer?